CMakeLists.txt

```diff
@@ -105,6 +105,17 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Generic")
     endif()
 endif()
 
+# Check external transport.
+SET(EXTERNAL_TRANSPORT_HEADER ""   CACHE PATH "External transports header file")
+SET(EXTERNAL_TRANSPORT ""          CACHE PATH "External transports code file")
+
+if(EXTERNAL_TRANSPORT)
+    set(EXTERNAL ON)
```

Comment:
Please use lowercase for internal variables.

Reply:
`EXTERNAL` moved to `external`